core/src/test/scala/com/evolutiongaming/kafka/flow/KafkaFlowSpec.scala

```diff
@@ -29,83 +22,102 @@ class KafkaFlowSpec extends FunSuite {
 
   test("happy path") {
 
-    val state = State(commands = List(
-      Command.ProduceRecords(ConsumerRecords.empty),
-      Command.ProduceRecords(ConsumerRecords.empty),
-      Command.ProduceRecords(consumerRecords(consumerRecord(partition = 0, offset = 0)))))
+    val state = State(commands =
+      List(
+        Command.ProduceRecords(ConsumerRecords.empty),
+        Command.ProduceRecords(ConsumerRecords.empty),
+        Command.ProduceRecords(consumerRecords(consumerRecord(partition = 0, offset = 0)))
+      )
+    )
 
     val program = ConstFixture.of(state) flatMap { f =>
       f.kafkaFlow.take(1).drain *> f.state.get
     }
     val result = program.unsafeRunSync()
 
-    assertEquals(result, State(
-      actions = List(
-        Action.ReleaseConsumer,
-        Action.ReleaseTopicFlow,
-        Action.Poll(consumerRecords(consumerRecord(partition = 0, offset = 0))),
-        Action.Poll(ConsumerRecords.empty),
-        Action.Poll(ConsumerRecords.empty),
-        Action.Subscribe(topic)(RebalanceListener.empty),
-        Action.AcquireTopicFlow,
-        Action.AcquireConsumer))
+    assertEquals(
+      result,
+      State(
+        actions = List(
+          Action.ReleaseConsumer,
+          Action.ReleaseTopicFlow,
+          Action.Poll(consumerRecords(consumerRecord(partition = 0, offset = 0))),
+          Action.Poll(ConsumerRecords.empty),
+          Action.Poll(ConsumerRecords.empty),
+          Action.Subscribe(NonEmptySet.of(topic))(RebalanceListener.empty),
```

Comment:
2.12.x build is complaining about something here:
https://github.com/evolution-gaming/kafka-flow/pull/130/checks?check_run_id=1595806851

Reply:
Fixed